docs/internal-documentation/maintenance/ReleaseProcess-test-planning.md

```diff
@@ -0,0 +1,18 @@
+Release process - Test Planning
+===============================
+
+The purpose of test planning is to raise the quality by making the release
+testing more structured, getting better coverage and finding more bugs.
+This gives us greater faith in our releases.
+
+## Release testing start up-meeting
+
+1. Compile a list of all pull requests with user-facing changes.
+
+2. For each test item, discuss how it could and should be tested.
+   Make sure there is a heading titled "How to test this PR" in the pull request
+   description.
```

Comment:
If not, should that be added in this process? If so, state that.

Reply:
IMHO we could deal with that situation outside of this process and continue this process once the situation has been dealt with.